js/LayersConfig.js

```diff
@@ -169,6 +172,21 @@ BR.LayersConfig = L.Class.extend({
         return result;
     },
 
+    createOverpassLayer: function (query) {
+        return new OverpassLayer({
+            overpassFrontend: this.overpassFrontend,
+            query: query,
+            minZoom: undefined,
```

Comment:
We really should set a minZoom and not allow users to query the whole world and put unnecessary load on the Overpass API. My suggestion would be `12`.